src/index.js

```diff
@@ -26,6 +29,7 @@ import { fetchPersistedQuery } from "./lib/fetchPersistedQuery"
 import { info } from "./lib/loggers"
 import { mergeSchemas } from "./lib/stitching/mergeSchemas"
 import { executableLewittSchema } from "./lib/stitching/lewitt/schema"
+import { executableStressSchema } from "./lib/stitching/exchange/schema"
```

Comment:
I'm guessing we want to rename all of these `stress` instances to `exchange`? If so here's one! `executableExchangeSchema`

Reply:
yes!! :)

Reply:
ooh found out whats going on, my vscode was set to match whole word when searching so wasn't catching any of these!